addons/survey/static/src/scss/survey_templates_results.scss

```diff
@@ -0,0 +1,25 @@
+.o_survey_results_topbar_filters {
+    .dropdown-toggle {
+        @include hover-focus {
+            text-decoration: none;
+        }
+        &:after {
+            margin-left: 1.2em;
+        }
+        .fa {
+            margin-right: .4em;
+        }
```

Comment:
That sounds very specific ^^

Usually we try to use bootstrap classes such as "ml-2" or "mr-1" when possible.
To get coherent style with the rest of the frontend.

Reply:
I did not know that.
Then add a small comment on top on the ".dropdown-toggle".

`// copied from event templates dropdown to get a coherent styling`

We usually start making common ressources / widget when there are 3 occurrences (or more) of the same concept.